assessment.sql

```diff
@@ -6,23 +6,60 @@ CREATE DATABASE "sql-crud-assessment";
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
 -- your code here
+CREATE TABLE licensees(
+id SERIAL PRIMARY KEY,
+surname CHARACTER VARYING,
+given_name CHARACTER VARYING,
+gender CHARACTER VARYING(1),
+height INTEGER,
+weight INTEGER,
+dob DATE,
+eye_color CHARACTER VARYING,
+hair_color CHARACTER VARYING
+);
+
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
 
 -- your code here
+INSERT INTO licensees(surname, given_name, gender, height, weight, dob, eye_color, hair_color)
+  VALUES(`Bishop', 'Caren', 'f', 63, 132, '1943-09-26', 'Brown', 'Black`);
```

Comment:
Those are backticks before `Bishop'` and after `'Black`. They're not like regular quotes. Other than that, this is a good INSERT. 9/10